test_scripts/Smoke/Resumption/commonStepsResumption.lua

```diff
@@ -0,0 +1,74 @@
+local commonStepsResumption = {}
```

Comment:
@LitvinenkoIra I think some other functions can be added to this module for avoid copy - paste in test scripts above.
Please think about this.